### ISSUE TITLE
Errors starting up service docker-flow-proxy

### ISSUE THREAD

**Issue by nvrnight** [vfarcic/docker-flow-proxy#460]
I'm using VirtualBox with 3 Linux/Ubuntu 17.10.1 VMs. 1 manager, 2 workers. I read your responses in the other issues about it being something with the overlay network, but I'm at a loss at what to do or how to troubleshoot that. I just started experimenting with docker yesterday.

docker service logs dfp

> user123@user123-VirtualBox:~/projects/myapp/razor2$ sudo docker service logs dfp
> dfp.1.qvocubh1l9aa@user123-VirtualBox    | 2018/04/24 18:17:53 Starting HAProxy
> dfp.1.qvocubh1l9aa@user123-VirtualBox    | 2018/04/24 18:17:54 Getting certs from http://198.105.254.24:8080/v1/docker-flow-proxy/certs
> dfp.1.m77rmf7oucjk@user123-VirtualBox    | 2018/04/24 18:18:17 Starting HAProxy
> dfp.1.m77rmf7oucjk@user123-VirtualBox    | 2018/04/24 18:18:17 Getting certs from http://198.105.254.24:8080/v1/docker-flow-proxy/certs

Commands run:

> docker network create --driver overlay proxy

> docker service create --name swarm-listener     --network proxy     --mount "type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock"     -e DF_NOTIFY_CREATE_SERVICE_URL=http://dfp:8080/v1/docker-flow-proxy/reconfigure     -e DF_NOTIFY_REMOVE_SERVICE_URL=http://dfp:8080/v1/docker-flow-proxy/remove     --constraint 'node.role==manager'     dockerflow/docker-flow-swarm-listener

> docker service create --name dfp     -p 80:80     -p 443:443     --network proxy -e SERVICE_NAME=dfp    -e LISTENER_ADDRESS=swarm-listener     dockerflow/docker-flow-proxy

**Comment by nvrnight** [comment]
I have started up other containers and am able to ping swarm-listener from them so I believe the internal DNS is working correctly. Is it possible the dfp service is trying to grab the certs using that url before the DNS configuration has been applied to the container?

I try pinging the dfp service from my container I used to ping swarm-listener and it says unknown host also which also makes me suspect that the container is not far enough along in its startup process to be in the swarm networks' DNS registration.

After further testing, the dfp service started up fine by using -e SERVICE_NAME=127.0.0.1 so that may add a bit more truth to my theory above.
